pipelines/_ingredients/VerifyDark.yaml

```diff
@@ -11,12 +11,15 @@ tasks:
       doFlat: false
       doApplyGains: false
       doFringe: false
+      doCalculateStatistics: true
+      isrStats.doCalibDistributionStatistics: true
```

Comment:
Depending on whether or not you change the field name in `ip_isr`, this would change.